### ISSUE TITLE
Can map source be configurable or even better would be if there is pop-up (instead of opening in browser) with pin on map

### ISSUE THREAD

**Issue by wisemonkey** [nomacs/nomacs#373]
Currently pressing on GPS-pin icon opens up browser with google maps.
It would be pretty good if source of map can be configurable for e.g. I would like to use http://maps.stamen.com/watercolor/#12/-33.8686/151.2094

Or if there is pop-up with map (with configurable source) that would be awesome

**Comment by diemmarkus** [comment]
good idea! I just don't know how to convert the coordinates for different maps. In your sample we would need to add ``-33.8686/151.2094`` to the url while google maps needs ``33°52'07.0"S+151°12'33.8"E``

**Comment by wisemonkey** [comment]
I'll try to see if I can figure out conversion process and update here, thanks

**Comment by wisemonkey** [comment]
I found: https://www.rapidtables.com/convert/number/degrees-minutes-seconds-to-degrees.html Will that help? Please let me know if further information is needed
